### PR TITLE
Changelog for 1.1.2

### PR DESCRIPTION
**Pull request by ikoevska**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,23 @@
...
+* [Implemented #587](https://github.com/NativeScript/nativescript-cli/issues/587): Added the `$ tns install` command. You can use it to quickly install all platforms and dependencies described in a `package.json` file.
+
+### Fixed
+
+* [Fixed #606](https://github.com/NativeScript/nativescript-cli/issues/606): The NativeScript CLI lets you run `<Platform>`-based commands on operating systems which do not support development for this platform. For example, you can run `$ tns build ios` on a Windows system.
+* [Fixed #601](https://github.com/NativeScript/nativescript-cli/issues/601): The NativeScript CLI does not treat the dependencies of devDependencies as devDependencies.
+* [Fixed #599](https://github.com/NativeScript/nativescript-cli/issues/599): The NativeScript CLI creates a `tmp` directory in the current directory when you run `$ tns run`.
```
Comment (Fatme) [inline CHANGELOG.md]: `tmp` folder was created from all commands.

**Comment by rosen-vladimirov** [comment]
:+1:

**Comment by dtopuzov** [comment]
:100:

**Comment by Fatme** [comment]
:+1:

**Pull request merged**